src/plugins/vis_type_timeseries/public/metrics_type.ts

```diff
@@ -25,11 +25,11 @@ export const metricsVisDefinition = {
   group: VisGroups.PROMOTED,
   visConfig: {
     defaults: {
-      id: '61ca57f0-469d-11e7-af02-69e470af7417',
+      id: uuid.v1(),
```

Comment:
We have chosen to use `import uuid from 'uuid/v4'` in Lens because it's fully random: uuid v1 is not a random ID.

Reply:
Yes Wylie, only here we are using hardcoded ids. Everywhere else we are using `uuid.v1()`
Nice feedback about v4, will use this instead 🙂